package.json

```diff
@@ -18,7 +18,7 @@
     "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
     "start-feature:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node-feature\" \"npm run watch-sass\"",
     "record-build-info": "node ./bin/record-build-info",
-    "lint": "eslint . --cache --max-warnings 0",
+    "lint": "eslint . --cache --max-warnings 0 && node_modules/.bin/shellcheck ./script/**/*[^utils] ./script/utils/**",
```

Comment:
I hope this is not breaking some unseen rule! I couldn't see any justification for it being a separate command in script/test but having it hear made sense? Happy to drop if not.

Reply:
I usually run `npm run lint -- --fix`, which I think this will break. Able to add something like `lint:fix", to do the same as above but with the first bit changed to `eslint . --cache --max-warnings 0 --fix`?